lib/install.js

```diff
@@ -96,6 +97,10 @@ function install(opts, cb) {
     tasks.push(setDriverFilePermissions.bind(null, fsPaths.firefox.installPath));
   }
 
+  if (fsPaths.edge) {
+    tasks.push(setDriverFilePermissions.bind(null, fsPaths.edge.installPath));
+  }
```

Comment:
Is this block needed? There isn't an existing similar one for IE. Or is there a bug that the IE block is missing?

That `setDriverFilePermissions()` ends up calling chmod which I'm not sure what happens on Windows.